sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2872,7 +2872,7 @@ def _provision(
                     wheel_hash=wheel_hash,
                     provision_record=provision_record,
                     custom_resource=resources_vars.get('custom_resources'),
-                    log_dir=self.log_dir)
+                    log_dir=os.path.expanduser(self.log_dir))
```

Comment:
How about we add it here instead? https://github.com/skypilot-org/skypilot/blob/bc3df3d159374d566c0e587fa612317ee661ab71/sky/provision/logging.py#L26

Reply:
Done. Thanks!